8.x/core-concepts/validation.md

```diff
@@ -2,15 +2,37 @@
 
 [[toc]]
 
-This document talks about checking the validity of requests that are sent from the client side to the server. The Larammerce platform is written based on the Laravel framework, so it uses the same validation rules with some modifications to make it easier.
+This document talks about checking the validity of requests sent from the client to the server. The Larammerce platform is written based on the Laravel framework, so it uses the same validation system with some modifications to make it easier.
 
 ### Validation in Laravel framework
 
-You can study the official Laravel documentation to learn more about this topic.*<sup>[1](#1)</sup>*
+Laravel provides several different approaches to validate your application's incoming data. It is most common to use the `validate` method available on all incoming HTTP requests. Laravel includes a wide variety of convenient validation rules that you may apply to data, even providing the ability to validate if values are unique in a given database table. You can study the official Laravel documentation to learn more about this topic.*<sup>[1](#1)</sup>*
 
 ### Validation in Larammerce platform
 
-Requests in the Larammerce platform are sent to the `Controllers` directory. For example in the `/path/to/larammerce-project/app/Http/Controllers/Admin/Api/ArticleController.php` there are some methods such as `index()`, `create()`, `store()`, ... . For each of these methods, multiple lines of repetitive codes must be written in order to validate the request. Instead, the `RuleMiddleware` is created in the `/path/to/larammerce-project/app/Http/Middleware/RuleMiddleware.php` file:
+Requests in the Larammerce platform are handed off to the router for dispatching. The router will dispatch the request to a route or controller, as well as run any route specific middleware. If the request passes through all of the matched route's assigned middleware, the route or controller method will be executed and the response returned by the route or controller method will be sent back through the route's chain of middleware.*<sup>[2](#2)</sup>*
+
+For example in the `ArticleController` there are some methods such as `index()`, `create()`, `store()`, ... . For each of these methods, multiple lines of repetitive codes must be written in order to validate the request. A given instance maybe as belows:
```

Comment:
For example ---> For example, 
 `ArticleController` --->    `ArticleController.php` file, there are....

(add file after the ArticleController and add a comma after the file and after the example...)

also put this path as comment on top of the code block in line 18.
// larammerce-project/app/Http/Controllers/Admin/Api/ArticleController.php

Reply:
@reyhane-rangamiz : As mentor said in the last session, we are not allowed to use "file" or "directory" or "path-of-the-file" here.

Reply:
@ShammasAmir   But I see you used the path over the document. the path would be good for clarity. like line 245. And also I see you didn't add the word **"file"** after the `ArticleController`. Also, why didn't you write ``ArticleController.php`?

Reply:
@reyhane-rangamiz : OK. Let's consult with the mentor.